activesupport/test/encrypted_file_test.rb

```diff
@@ -56,4 +56,22 @@ class EncryptedFileTest < ActiveSupport::TestCase
       ).read
     end
   end
+
+  test "keep the symbolic link" do
+    @encrypted_file.write(@content)
+
+    link_path = File.join(Dir.tmpdir, "link_content.txt.enc")
+    File.symlink(@encrypted_file.content_path, link_path)
+
+    begin
+      link_encrypted_file = ActiveSupport::EncryptedFile.new(
+        content_path: link_path, key_path: @key_path, env_key: "CONTENT_KEY", raise_if_missing_key: true
+      )
+      link_encrypted_file.write(@content)
+
+      assert File.symlink?(link_path)
```

Comment:
It’s worth asserting that the new contents were written to the realpath:

```ruby
assert_equal @content, @encrypted_file.read
```

Reply:
I'll add an assert to test writing to realpath.